libindi/libs/indibase/alignment/MathPlugin.cpp

```diff
@@ -18,5 +18,59 @@ bool MathPlugin::Initialise(InMemoryDatabase *pInMemoryDatabase)
     return true;
 }
 
+std::string MathPlugin::GetInternalDataRepresentation(std::string PluginDisplayName)
```

Comment:
I'm not a huge C++ expert but std::string is defined in <string> but Common.h only includes <cstring>.
Is this the reason? Strange that gcc and clang4/5 on linux did not complain...

Reply:
Neither do I... On my linux gcc `<cstring>` includes `<string>` so that should do the job. I try with including `<string>` and `<cstdio>` too for `snprintf`.